dizoo/classic_control/cartpole/config/cartpole_acer_config.py

```diff
@@ -34,6 +34,7 @@
             # (float) clip ratio of importance weights
             trust_region=True,
             c_clip_ratio=10,
+            # load_path='/Users/simon/Documents/sensetime/default_experiment/ckpt/iteration_800.pth.tar'
```

Comment:
if changes this line to # load_path=<model ckpt PATH>  rather than sepecifies the certain path.